src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUM_BALLS = 3;
+
     public static void main(String[] args) {
+
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < NUM_BALLS; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball);
```

Comment:
Expression lottery.getRandomBall() is rather clear, short and used one time. 
See no benefit from separate variable creation
```suggestion
            System.out.println(lottery.getRandomBall());
```